images/php/config/laravel/main.tf

```diff
@@ -0,0 +1,19 @@
+terraform {
+  required_providers {
+    apko = { source = "chainguard-dev/apko" }
+  }
+}
+
+variable "extra_packages" {
+  description = "The additional packages to install (e.g. php-fpm)."
+  default     = ["php", "php-fpm", "php-ctype", "php-xml", "php-dom", "php-simplexml", "php-fileinfo"]
```

Comment:
```suggestion
  default     = ["curl", "ca-certificates", "php", "php-fpm", "php-ctype", "php-curl", "php-dom", "php-fileinfo", "php-iconv", "php-mbstring", "php-mysqlnd", "php-openssl", "php-phar", "php-pdo", "php-pdo_sqlite", "php-pdo_mysql", "php-sodium", "php-simplexml", "php-xml"]
```

Reply:
Putting the packages here means they can be removed from the apko.yaml file. Doing it this way will let us create other images that import this module define their own sets of `extra_packages` without running into conflicts with different versions, provides, requires etc.